src/panels/NightSkyPanel/tabs/NightSkyLocationTab.tsx

```diff
@@ -0,0 +1,155 @@
+import {
+  BackgroundImage,
+  Button,
+  Divider,
+  Group,
+  Image,
+  Paper,
+  Stack,
+  Text
+} from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { useSubscribeToCamera } from '@/hooks/topicSubscriptions';
+import { LocationPinIcon } from '@/icons/icons';
+import { useAppSelector } from '@/redux/hooks';
+
+export function NightSkyLocationTab() {
+  const luaApi = useOpenSpaceApi();
+  useSubscribeToCamera();
+
+  const {
+    latitude: currentLat,
+    longitude: currentLong,
+    altitude: currentAlt
+  } = useAppSelector((state) => {
+    return state.camera;
+  });
+
+  function dotPosition() {
+    if (currentLong && currentLat) {
+      return { x: ((currentLong + 180) / 360) * 100, y: ((90 - currentLat) / 180) * 100 };
+    } else {
+      return { x: 0, y: 0 };
+    }
+  }
+
+  function look(direction: string): void {
+    luaApi?.action.triggerAction('os.nightsky.Looking' + direction);
+  }
+
+  return (
+    <>
+      <Group justify={'center'}>
+        <Text size={'md'}>
+          Globe position - Latitude: {currentLat?.toFixed(2)}, Longitude:{' '}
+          {currentLong?.toFixed(2)}, Altitude: {currentAlt?.toFixed(2)}m
+        </Text>
+        <BackgroundImage w={300} h={150} src={'eqcy.png'}>
+          <Image
+            src={'icon.png'}
+            style={{
+              width: '10px',
+              position: 'relative',
+              left: dotPosition().x - 0.1666666666 + '%',
+              top: dotPosition().y - 0.1666666666 + '%'
+            }}
+          ></Image>
+        </BackgroundImage>
+      </Group>
```

Comment:
```suggestion
      <Stack>
        <Text size={'md'}>
          Globe position - Latitude: {currentLat?.toFixed(2)}, Longitude:{' '}
          {currentLong?.toFixed(2)}, Altitude: {currentAlt?.toFixed(2)}m
        </Text>
        <BackgroundImage w={300} h={150} src={'eqcy.png'}>
          <Image
            src={'icon.png'}
            style={{
              width: '10px',
              position: 'relative',
              left: dotPosition().x - 0.1666666666 + '%',
              top: dotPosition().y - 0.1666666666 + '%'
            }}
          ></Image>
        </BackgroundImage>
      </Stack>
```
When they were grouped the image would jitter as the text was changing